src/store/plugins/reproject-everything-on-projection-change.plugin.js

```diff
@@ -32,9 +32,6 @@ const reprojectEverythingOnProjectionChangePlugin = (store) => {
                 const reprojectCoordinates = (coordinates = []) =>
                     proj4(oldProjection.epsg, newProjection.epsg, coordinates)
 
-                // reprojecting the center of the map
-                store.dispatch('setCenter', reprojectCoordinates(state.position.center))
```

Comment:
I would suggest to rename this plugin as it only reproject selected features and not everything.